src/core/control/Control.cpp

```diff
@@ -1152,7 +1153,15 @@ void Control::selectFillAlpha(bool pen) {
 }
 
 void Control::selectPageMoveTo() {
-    std::cout << "Page Move entry not implemented yet" << std::endl;
+    MovePageDialog dlg(gladeSearchPath, getCurrentPageNo(), int(this->doc->getPageCount()));
```

Comment:
Is the int cast needed? `getCurrentPageNo` returns `size_t` likewise `getPageCount`.

Reply:
No, especially not if I change the page numbers in `MovePageDialog` to `size_t` anyways